<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontAdjuster</class>
 <widget class="QMainWindow" name="FontAdjuster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font Size and Color Adjuster</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout" stretch="1,1,1,1">
    
    <item>
     <widget class="QLabel" name="labelNIM">
      <property name="text">
       <string>F1D022122</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimumHeight">
       <number>100</number>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>

    <!-- Font Size Slider -->
    <item>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="labelFontSizeTitle">
        <property name="text">
         <string>Font Size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="labelFontMin">
          <property name="text">
           <string>20</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderFontSize">
          <property name="minimum">
           <number>20</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFontMax">
          <property name="text">
           <string>60</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>

    <!-- Background Color Slider -->
    <item>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="labelBgColorTitle">
        <property name="text">
         <string>Background Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderBgColor">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel">
          <property name="text">
           <string>255</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>

    <!-- Font Color Slider -->
    <item>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="labelFontColorTitle">
        <property name="text">
         <string>Font Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderFontColor">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel">
          <property name="text">
           <string>255</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>

    <!-- Nama & NIM -->
    <item>
     <widget class="QLabel" name="labelNamaNIM">
      <property name="text">
       <string>Hendra Ahmad Yani - F1D022122</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignBottom</set>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray; font-size: 10pt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
